gapil/langsvr/analyze.go

```diff
@@ -250,10 +251,12 @@ func (a *analyzer) doAnalysis(
 		defer shutdown(ctx)
 		events := &task.Events{}
 		executor := task.Batch(pool, events)
-
+		mt := sync.Mutex{}
 		for path, da := range das {
-			path, da, ctx := path, da, log.V{"file": path}.Bind(ctx)
+			path, da, _ := path, da, log.V{"file": path}.Bind(ctx)
```

Comment:
What's the point of the 3rd assignment now?

Reply:
Good point!